packages/instantsearch.js/src/lib/InstantSearch.ts

```diff
@@ -341,10 +350,12 @@ See documentation: ${createDocumentationLink({
     this.future = future;
     this.insightsClient = insightsClient;
     this.indexName = indexName;
+    this.compositionID = compositionID;
     this.helper = null;
     this.mainHelper = null;
     this.mainIndex = index({
-      indexName,
+      // we use an index widget to render compositions
```

Comment:
clearer comment to explain that we're able to do this because there's only one composition index allowed for now.

Reply:
✅